metrics/wait.io.table/table.go

```diff
@@ -197,6 +198,7 @@ LEVEL:
 		} else {
 			o.truncateTimeout = 250 * time.Millisecond // default
 		}
+		o.metricType = blip.CUMULATIVE_COUNTER
```

Comment:
I think it's the opposite, if you truncate after each collection you are going to get Delta on the next collection. I could move these into the If Block above to make it clearer, but it will work as is as well, because putting `o.metricType = blip.CUMULATIVE_COUNTER` here is equivalent to `if o.truncate == false` as I override it with Delta inside `if o.truncate` right after this assignment. But I will move anyway to make it clearer.